system/doc/efficiency_guide/processes.xmlsrc

```diff
@@ -268,87 +271,98 @@ cross_function_receive(Ref) ->
+    The following function does not build the tuple every time
+    it is called (only to have it discarded the next time the garbage
+    collector was run), but the tuple is located in the module's
+    literal pool:</p>
 
     <p><em>DO</em></p>
-      <code type="erl">
+    <code type="erl"><![CDATA[
 days_in_month(M) ->
-    element(M, {31,28,31,30,31,30,31,31,30,31,30,31}).</code>
-
-      <p>But if a constant is sent to another process (or stored in
-      an Ets table), it is <em>copied</em>.
-      The reason is that the runtime system must be able
-      to keep track of all references to constants to unload code
-      containing constants properly. (When the code is unloaded,
-      the constants are copied to the heap of the processes that refer
-      to them.) The copying of constants might be eliminated in a future
-      Erlang/OTP release.</p>
-    </section>
+    element(M, {31,28,31,30,31,30,31,31,30,31,30,31}).]]></code>
 
-    <section>
-      <title>Loss of Sharing</title>
+    <p>If a literal, or a term that contains a literal, is inserted
+    into an Ets table, it is <em>copied</em>. The reason is that the
+    module containing the literal can be unloaded in the future.</p>
 
-      <p>Shared subterms are <em>not</em> preserved in the following
-      cases:</p>
-      <list type="bulleted">
-	<item>When a term is sent to another process</item>
-	<item>When a term is passed as the initial process arguments in
-	the <c>spawn</c> call</item>
-	<item>When a term is stored in an Ets table</item>
-      </list>
-      <p>That is an optimization. Most applications do not send messages
-      with shared subterms.</p>
+    <p>When a literal is sent to another process, it is <em>not</em>
+    copied. When a module holding a literal is unloaded, the literal
+    will be copied to the heap of all processes that hold references
+    to that literal.</p>
+
+    <p>There also exists a global literal pool that is managed by the
+    <seeerl marker="erts:persistent_term">persistent_term</seeerl>
+    module.</p>
 
-      <p>The following example shows how a shared subterm can be created:</p>
+    <p>By default, 1 GB of virtual address space is reserved for all
+    literal pools (in BEAM code and persistent terms). The amount of
+    virtual address space reserved for literals can be changed by
+    using the <seecref marker="erts:erts_alloc#MIscs"><c>+MIscs
+    option</c></seecref> when starting the emulator.</p>
 
-      <code type="erl">
-kilo_byte() ->
-    kilo_byte(10, [42]).
+    <p>Here is an example how the reserved virtual address space for
+    literals can be raised to 2 GB (2048 MB):</p>
 
-kilo_byte(0, Acc) ->
-    Acc;
-kilo_byte(N, Acc) ->
-    kilo_byte(N-1, [Acc|Acc]).</code>
+    <pre>
+    erl +MIscs 2048</pre>
+  </section>
+
+  <section>
+    <marker id="loss-of-sharing"></marker>
+    <title>Loss of Sharing</title>
+
+    <p>Shared subterms are <em>not</em> preserved in the following
+    cases:</p>
```

Comment:
Here I think it would be nice to start off with a really simple example of shared subterms to explain what it is. Like `{SubTerm, SubTerm}`.

Reply:
Done.